src/psyclone/tests/nemo/transformations/profiling/nemo_profile_test.py

```diff
@@ -43,6 +43,7 @@
 from __future__ import absolute_import
 import pytest
 from fparser.common.readfortran import FortranStringReader
+from fparser.two.symbol_table import SYMBOL_TABLES
```

Comment:
Please update the Copyright modification year to 2022.

Reply:
Oops.